app/positionkit.h

```diff
@@ -192,11 +230,25 @@ class PositionKit : public QObject
      */
     Q_INVOKABLE void useSimulatedLocation( double longitude, double latitude, double radius );
 
+    // Starts receiving updates from GPS sources
+    void startUpdates();
+
+    // Stops receiving updates from GPS sources
+    void stopUpdates();
+
     /**
      * Use real GPS source (not simulated)
      */
     Q_INVOKABLE void useGpsLocation();
 
+    const QDateTime &lastGPSRead() const;
+
+    QGeoSatelliteInfoSource *satelliteSource() const;
```

Comment:
what properties do you read from it? do we need to expose the source outside of the position kit? also , why the satellite source is separate from "source"... ?

Reply:
It is used in tests